webpack.config.js

```diff
@@ -2,123 +2,146 @@
 /* eslint-disable import/no-nodejs-modules*/
 const path = require('path');
 const glob = require('glob');
+const rimraf = require('rimraf');
 const webpack = require('webpack');
 const CleanObsoleteChunksPlugin = require('webpack-clean-obsolete-chunks');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const {styles} = require('@ckeditor/ckeditor5-dev-utils'); // eslint-disable-line import/no-extraneous-dependencies
 
-const entries = glob.sync(
-    path.resolve(__dirname, 'src/Sulu/Bundle/*/Resources/js/index.js') // eslint-disable-line no-undef
-);
-const entriesCount = entries.length;
+module.exports = (env, argv) => { // eslint-disable-line no-undef
+    let publicDir = 'public';
+    const basePath = env && env.base_path ? env.base_path : 'build/admin';
 
-entries.unshift('core-js/fn/array/includes');
-entries.unshift('core-js/fn/array/find-index');
-entries.unshift('core-js/fn/array/fill');
-entries.unshift('core-js/fn/array/from');
-entries.unshift('core-js/fn/promise');
-entries.unshift('core-js/fn/symbol');
-entries.unshift('whatwg-fetch');
-entries.unshift('url-search-params-polyfill');
+    const composerConfig = require(path.resolve('composer.json')); // eslint-disable-line import/no-dynamic-require
```

Comment:
Isn't that the wrong `composer.json` in case it is run in sulu-minimal? :thinking:

Reply:
no else path.resolve('public')  before would also be false.